pkg/rancher/rancher.go

```diff
@@ -253,12 +253,14 @@ func (r *Rancher) Start(ctx context.Context) error {
 	}
 
 	if features.MCM.Enabled() {
+		// adds system-charts repo
```

Comment:
This comment might add some confusion to the users since this adding system-chats repo is just one of many things this function does.
```suggestion
```

Reply:
That's true, still I found it surprising that this happens so early.